e2etest/bgp_test.go

```diff
@@ -442,38 +462,56 @@ var _ = ginkgo.Describe("BGP", func() {
 					Addresses: []string{
 						"fc00:f853:0ccd:e799::/124",
 					},
-				}}, "ipv6",
+				}}, "ipv6", testservice.TrafficPolicyCluster,
+			),
+			table.Entry("DUALSTACK - test AddressPool defined by address range", []addressPool{
+				{
+					Name:     "bgp-test",
+					Protocol: BGP,
+					Addresses: []string{
+						"192.168.10.0-192.168.10.18",
+						"fc00:f853:0ccd:e799::0-fc00:f853:0ccd:e799::18",
+					},
+				}}, "ipv4", testservice.TrafficPolicyCluster,
```

Comment:
why we passing ipv4 here and below for dual-stack

Reply:
not necessarily. The ipfamily now drives only how the bgp session is established, which is orthogonal to the family of the service.